src/BidiStream.php

```diff
@@ -144,6 +168,22 @@ public function read()
                 throw ApiException::createFromStdClass($status);
             }
         }
+
+        if ($this->logger) {
+            $responseEvent = new RpcLogEvent();
+
+            $responseEvent->headers = (is_null($result)) ? $this->call->getMetadata() : null;
+            $responseEvent->status = (is_null($result)) ? $status->code : null;
+            $responseEvent->processId = (int) getmypid();
+            $responseEvent->requestId = crc32((string) spl_object_id($this) . getmypid());
+
+            if ($result && $result instanceof Message) {
```

Comment:
not necessary to check if `$result` exists first - if result is null this will be false already
```suggestion
            if ($result instanceof Message) {
```